source/includes/administration/_custom_ui_settings.md

```diff
@@ -0,0 +1,246 @@
+### Custom UI Settings
+
+<!-------------------- FIND CUSTOM UI SETTINGS -------------------->
+
+#### Find custom ui settings associated to an organization
+
+`GET /reseller/settings/custom_ui/find?organizationId=:id`
+
+Retrieve the custom ui settings associated to an organization. If the `organizationId` is omitted, the authenticated user's organization will be used.
+
+```shell
+# Retrieve the custom ui settings
+curl "https://cloudmc_endpoint/api/v2/reseller/settings/custom_ui/find?organizationId=10572c3d-16e5-450f-8af8-a01e50dc52d4" \
+   -H "MC-Api-Key: your_api_key"
+```
+
+> The above command returns JSON structured like this:
+
+```json
+{
+  "data": {
+    "organization": {
+      "id": "10572c3d-16e5-450f-8af8-a01e50dc52d4"
+    },
+    "id": "d785ffcb-9b03-478d-a49b-52a2ccedf1b8",
+    "version": 1,
+    "enabled": true,
+    "customUIRootUrl": "https://newui.example.com",
+    "domainToken": "example.com",
+    "applyTags": [],
+    "denyTags": [],
+    "redirectRoles": []
+  }
+}
+```
+
+Retrieve the custom UI settings associated to an organization.
+
+Attributes | &nbsp;
+---------- | -----------
+`id`<br/>*UUID* | The configured custom UI settings' id.
+`organization.id`<br/>*UUID* | The organization id that the custom UI settings are linked to. It cannot be changed.
+`version`<br/>*integer* | The custom UI settings version.
+`enabled`<br/>*boolean* | If the custom ui redirection is enabled or not.
+`customUIRootUrl`<br/>*string* | The base root that the user will be redirected to once the user is login.
+`domainToken`<br/>*string* | The domain name that will be used in the authentication cookie. The cookie must be acceptable by both the reseller organization custom domain and the url provided in this configuration.
+`applyTags`<br/>*Array[string]* | The list of organization tags that will redirect the user of sub-organizations to the custom ui if they match.
+`denyTags`<br/>*Array[string]* | The list of organization tags that will not redirect the user of sub-organizations to the custom ui if they match.
```

Comment:
Same here, should be custom **UI** to be consistent (and in a few other places in other sections)

Reply:
e1fe79e8e800d38760fbeeac4d2ed4a17474aa49